src/actions/guides/database/querying-deleting.cr

```diff
@@ -668,6 +668,51 @@ class Guides::Database::QueryingDeleting < GuideAction
     > Avram is designed to be type-safe. You should use caution when using the non type-safe methods,
     > or raw SQL.
 
+    ## Reseting Queries
+
+    If you need to remove parts of the SQL query after the query has been built, Avram gives you
+    a few reset methods for that.
+
+    ### Reset where
+
+    The `reset_where` method takes a block where you call the name of the column you want to remove
+    from your query.
+
+    ```crystal
+    user_query = UserQuery.new.name("Billy").signed_up.lt(2.days.ago)
+
+    # This will only remove the `name = 'Billy'` from the WHERE clause
+    # SELECT * FROM users WHERE signed_up < ?
```

Comment:
I was a bit confused about this because I didn’t see the name in the SQL. Then I got it after a bit :P

Maybe we could help slow people like me by first showing what the sql would be before the reset. And SQL after. That might clear it up. Thoughts?